elasticsearch-model/gemfiles/3.0.gemfile

```diff
@@ -9,4 +9,4 @@ gemspec path: '../'
 
 gem 'activemodel',  '>= 3.0'
 gem 'activerecord', '~> 3.2'
-gem 'mongoid',      '>= 3.0'
+gem 'mongoid',      '~> 3.0'
```

Comment:
Without this change, both gemfiles will test against Mongoid 4.x.

This change is needed so `3.0.gemfile` tests against Mongoid 3.x and AR 3.x, and `4.0.gemfile` tests agains Mongoid 4.x and AR 4.x.